.eleventy.js

```diff
@@ -1,53 +1,104 @@
+/*
+Copyright the Inclusive Learning Design Handbook copyright holders.
+
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-project/handbook.floeproject.org/raw/main/AUTHORS.md.
+
+Licensed under the New BSD license. You may not use this file except in compliance with this License.
+
+You may obtain a copy of the New BSD License at
+https://github.com/fluid-project/handbook.floeproject.org/raw/main/LICENSE.md.
+*/
+
 "use strict";
 
 const fs = require("fs");
-const fluidPlugin = require("@fluid-project/eleventy-plugin-fluid");
+
+const fluidPlugin = require("eleventy-plugin-fluid");
+const { EleventyRenderPlugin } = require("@11ty/eleventy");
+const MarkdownIt = require("markdown-it");
 const navigationPlugin = require("@11ty/eleventy-navigation");
+const rssPlugin = require("@11ty/eleventy-plugin-rss");
 const syntaxHighlightPlugin = require("@11ty/eleventy-plugin-syntaxhighlight");
+
+// Import transforms
+const htmlMinTransform = require("./src/transforms/html-min-transform.js");
 const parseTransform = require("./src/transforms/parse-transform.js");
 
+// Import data files
+const siteConfig = require("./src/_data/config.json");
+const getResourceLinks = require("./src/utils/getResourceLinks.js");
+
 module.exports = function (config) {
     config.setUseGitIgnore(false);
 
-    // Layouts
-    config.addLayoutAlias("default", "layouts/default.njk");
-
-    // Plugins
-    config.addPlugin(fluidPlugin);
-    config.addPlugin(navigationPlugin);
-    config.addPlugin(syntaxHighlightPlugin);
+    // Filters
+    const md = new MarkdownIt({
+        html: true,
+        quotes: "“”‘’"
+    });
+    config.addFilter("markdown", (content) => {
+        return md.render(content);
+    });
 
     // Transforms
+    config.addTransform("htmlmin", htmlMinTransform);
     config.addTransform("parse", parseTransform);
 
     // Passthrough copy
     config.addPassthroughCopy("src/_redirects");
-    config.addPassthroughCopy({"src/assets/images": "assets/images"});
+    config.addPassthroughCopy({"src/admin/config.yml": "admin/config.yml"});
     config.addPassthroughCopy({"src/assets/fonts": "assets/fonts"});
-    config.addPassthroughCopy({"node_modules/docs-core/src/static/css": "assets/styles"});
-    config.addPassthroughCopy({"node_modules/docs-core/src/static/lib": "lib"});
+    config.addPassthroughCopy({"src/assets/icons": "/"});
+    config.addPassthroughCopy({"src/assets/images": "assets/images"});
+
+    // Plugins
+    config.addPlugin(EleventyRenderPlugin);
+    config.addPlugin(fluidPlugin);
+    config.addPlugin(navigationPlugin);
+    config.addPlugin(rssPlugin);
```

Comment:
What was the discussion result regarding whether the RSS feed code should be kept in the repo?

Reply:
We decided that it did not make sense in this context; the ILDH is more of a book to which we add chapters occasionally rather than a news site in the same vein as, for example, the Floe Project site. I'll remove the code associated with this functionality